scripts/eurostat/life_expectancy/preprocess.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+import re
+
+PATH = 'demo_r_mlifexp.tsv'
+
+def nuts_to_iso(data):
+    """Convert 2 letter nuts code of countries to ISO Alpha-3 code"""
+    ISO_2_TO_3_PATH = ('https://gist.githubusercontent.com/tadast/8827699/raw/'
+                       '3cd639fa34eec5067080a61c69e3ae25e3076abb/'
+                       'countries_codes_and_coordinates.csv')
+    codes = pd.read_csv(ISO_2_TO_3_PATH)
```

Comment:
Pandas's documentation for how read.csv signals a failure does not exist.  We will assume it throws an exception.  No change is needed.